src/doom/st_stuff.c

```diff
@@ -451,6 +451,7 @@ cheatseq_t cheat_showfps2 = CHEAT("idrate", 0); // [crispy] PrBoom+
 cheatseq_t cheat_goobers = CHEAT("goobers", 0);
 cheatseq_t cheat_version = CHEAT("version", 0); // [crispy] Russian Doom
 cheatseq_t cheat_skill = CHEAT("skill", 0);
+cheatseq_t cheat_snow = CHEAT("snow", 0);
```

Comment:
I like the idea of turning it in to a cheat code. However, "snow" is a but generic, isn't it? Put some music into it, how about "letitsnow"? :smile: